tools/maxquant/maxquant.xml

```diff
@@ -309,51 +285,22 @@ sample:$j
     </macro>
   </macros>
   <inputs>
-    <conditional name="analysis_type">
-      <param name="type" type="select" label="Analysis Type" help="The wrapper has not yet implemented multiple groups with different parameters">
-        <option value="sg_fractions" selected="true">Single Group - Fractions of One Sample</option>
-        <option value="sg_samples">Single Group - Single File Samples</option>
-        <option value="sg_both">Single Group - Multiple Fractions and Samples</option>
-        <option value="mgs_fractions">Multi-Group (Identical Parameters) - Fractions of One Sample</option>
-        <option value="mgs_samples">Multi-Group (Identical Parameters) - Single File Samples</option>
-        <option value="mgs_both">Multi-Group (Identical Parameters) - Multiple Fractions and Samples</option>
-      </param>
-      <when value="mgs_fractions">
-        <repeat name="groups" title="Group">
-          <expand macro="input_param" />
-        </repeat>
-	      <expand macro="group_params" />
-      </when>
-      <when value="mgs_samples">
-        <repeat name="groups" title="Group">
-          <expand macro="input_param" />
-        </repeat>
-        <expand macro="group_params" />
-      </when>
-      <when value="mgs_both">
-        <repeat name="groups" title="Group">
-          <repeat name="samples" title="Sample">
-            <expand macro="input_param" />
-          </repeat>
-        </repeat>
-        <expand macro="group_params" />
-      </when>
-      <when value="sg_fractions">
-        <expand macro="input_param" />
-        <expand macro="group_params" />
-      </when>
-      <when value="sg_samples">
-        <expand macro="input_param" />
-        <expand macro="group_params" />
-      </when>
-      <when value="sg_both">
-        <repeat name="samples" title="Sample">
-          <expand macro="input_param" />
-        </repeat>
-        <expand macro="group_params" />
-      </when>
-    </conditional>
+    <repeat name="sample" title="Sample">
```

Comment:
In fact I did not remove completely the possibilitie of selecting several parameter groups. To do that (and maintaining this interface of select raw data that we changed) I'd have to change the xml so the user could select the parameters for every group, and I'd also have to change the wrapper to take this different parameters for each group. But I am planning to do that later, meanwhile we want to let the usual features working properly.
Sorry but I didn't understand what you mean about MSI. What is it?